### ISSUE TITLE
Back button consistency

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#142]
Back button should be clearly labelled on all screens and read by voice over as this is a navigation issue. Thus, any accessible user should not get stuck at any given point while navigating within the app.

Note: Voiceover is missing the back button on Manage subjects screen.

**Comment by ForumG-SC** [comment]
Done
